sensor_msgs/PointCloud2 point_cloud
string frame_id
---
sensor_msgs/PointCloud2 point_cloud